### PR TITLE
Arc 1354 add ff to write to cryptor

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Add logic to use FF to write to new secret column only.

**Why**
To deprecate the old col

**Added feature flags**
"write-only-to-new-cryptor-shared-secret"

**Affected issues**  
ARC-1354

**How has this been tested?**  
Unit test.
Stage (WIP)

**Whats Next?**
Rollout the FF, and then tidy up.

### CODE REVIEW

File: src/github/client/github-client.test.ts [hunk 1]
```diff
@@ -50,13 +50,13 @@ describe("GitHub Client", () => {
 		expect(outboundProxyConfig.httpsAgent).toBeUndefined();
 	});
 
-	describe('config object', () => {
-		const TEST_API_URL = 'http://api.myBaseUrl.com';
-		const TEST_GRAPHQL_URL = 'http://graphql.myBaseUrl.com';
+	describe("config object", () => {
```
Comment (gxueatlassian) [inline src/github/client/github-client.test.ts:53]: I do this before commit `yarn lint:ts --fix`, not sure why there was `'`, but seems the linter always change it to `"`

File: src/github/issue-comment.test.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ describe("Issue Comment Webhook", () => {
 		await Installation.create({
 			jiraHost,
 			clientKey: "client-key",
-			sharedSecret: "shared-secret"
+			encryptedSharedSecret: "shared-secret"
```
Comment (gxueatlassian) [inline src/github/issue-comment.test.ts:24]: So these tests are not directly related to the change.
But instead of mocking FF to true and/or false for all these test, I simply convert it to using the new col only, so that it is easier and closer to the final state we want.

File: src/models/github-server-app.ts [hunk 3]
```diff
@@ -38,11 +38,6 @@ export class GitHubServerApp extends EncryptedModel {
 	}
 
 	async getEncryptContext() {
-		//For example: we can call database to fetch sharedSecret for use as EncryptionContext
```
Comment (gxueatlassian) [inline src/models/github-server-app.ts:41]: No idea that the he** is this block of comments? lol

**Review by mboudreau — APPROVED** [review]
